server/controllers/users-controller.js

```diff
@@ -0,0 +1,40 @@
+import { successResponse, generateToken, errorResponse } from '../utils/helpers';
+import { User } from '../models';
+
+/**
+ * The controllers for users route
+ *
+ * @class UsersController
+ */
+class UsersController {
+  /**
+     * User registration controller
+     *
+     * @static
+     * @param {*} req
+     * @param {*} res
+     * @param {Function} next
+     * @memberof UsersController
+     * @returns {undefined}
+     */
+  static register(req, res) {
+    const { user } = req.body;
```

Comment:
The fields of interest should ideally be directly on the `request.body` object.

Reply:
Agreed

Reply:
From the `README` file the field of interest is `user` 👇 
```
{
  "user":{
    "username": "Jacob",
    "email": "jake@jake.jake",
    "password": "jakejake"
  }
}
```